test/hca-rjsf/config/veteranInformation.unit.spec.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import ReactTestUtils from 'react-dom/test-utils';
+
+import { DefinitionTester } from '../../util/schemaform-utils.jsx';
+import formConfig from '../../../src/js/hca-rjsf/config/form.js';
+
+describe('HCA veteranInformation', () => {
+  it('should render veteranInformation page', () => {
```

Comment:
Same comment as on the other PR, I think we want some tests around submitting the form and checking that the right number of required errors happen.